MantidQt/CustomInterfaces/test/ReflSettingsTabPresenterTest.h

```diff
@@ -65,6 +191,33 @@ class ReflSettingsTabPresenterTest : public CxxTest::TestSuite {
     presenter.getStitchOptions();
     TS_ASSERT(Mock::VerifyAndClearExpectations(&mockView));
   }
+
+  void testExperimentDefaults() {
+    MockSettingsTabView mockView;
+    ReflSettingsTabPresenter presenter(&mockView);
+
+    std::vector<std::string> defaults = {"PointDetectorAnalysis", "None", "1"};
+
+    EXPECT_CALL(mockView, setExpDefaults(defaults)).Times(1);
+    presenter.notify(IReflSettingsTabPresenter::ExpDefaultsFlag);
```

Comment:
I suggest adding here the following line:

```
TS_ASSERT(Mock::VerifyAndClearExpectations(&mockView));
```

that way we can verify the expectations on our `mockView` at this point. Please add this to all tests in `ReflSettingsTabPresenterTest` that are missing it.